spec/data-model/README.md

```diff
@@ -110,18 +110,16 @@ interface CatchallKey {
 ## Patterns
 
 Each `Pattern` represents a linear sequence, without selectors.
```

Comment:
You might want to adjust this sentence also. It might a relic of earlier times, before the addition of pattern delimiters. Perhaps:

```suggestion
Each `Pattern` is a standalone linear sequence of text and placeholders
that represents the output of a message.
```